org.eclipse.winery.compliance/src/main/java/org/eclipse/winery/compliance/matching/TOSCAComplianceRuleMatcher.java

```diff
@@ -30,88 +30,88 @@
 
 public class TOSCAComplianceRuleMatcher implements ITOSCAMatcher {
```

Comment:
Our naming convention is that we always use camel case - even for abbreveations to enhance readability. So Please rename this class to `ToscaComplianceRuleMatcher`.

In this case, is it possible to also rename all other classes which do not follow this convention in the compliance project? (`ITOSCAMatcher`, `TOSCANode`, ...)